proxy/httpproxy/reverse.go

```diff
@@ -75,10 +75,14 @@ func (p *reverseProxy) ServeHTTP(rw http.ResponseWriter, clientreq *http.Request
 		proxybody, err = ioutil.ReadAll(clientreq.Body)
 		if err != nil {
 			msg := fmt.Sprintf("failed to read request body: %v", err)
-			plog.Println(msg)
+			p.lg.Info(msg, zap.Error(err))
```

Comment:
@jingyih thanks, lgtm except one comment here that, the `msg` already has `err` so we shouldn't repeat it, and just use `p.lg.Info(msg)` or remove `err` from the `msg`? Thanks!

Reply:
Thanks! Addressed.

Reply:
Thanks @jingyih